akka-stream/src/main/scala/akka/stream/scaladsl/Sink.scala

```diff
@@ -98,6 +98,16 @@ object Sink {
   def headOption[T]: Sink[T, Future[Option[T]]] = new Sink(new HeadOptionSink[T](DefaultAttributes.headSink, shape("HeadOptionSink")))
 
   /**
+   * TODO: document usage
+   *
+   */
+  def toSeq[T]: Sink[T, Future[Seq[T]]] = {
```

Comment:
remember to mirror it in https://github.com/akka/akka/blob/release-2.3-dev/akka-stream/src/main/scala/akka/stream/javadsl/Sink.scala afterwards (and tests :))